python/graphscope/client/session.py

```diff
@@ -204,6 +205,27 @@ def __init__(
 
             k8s_coordinator_mem (str, optional): Minimum number of memory request for coordinator pod. Defaults to '4Gi'.
 
+            k8s_volumes (dict, optional): A dict of k8s volume which represents a directory containing data, accessible to the
+                containers in a pod. Defaults to {}. Only 'hostPath' supported yet. For example, we can mount host path with:
+
+                k8s_volumes = {
+                    "my-data": {
+                        "type": "hostPath",
+                        "field": {
+                            "path": "<path>",
+                            "type": "Directory"
+                        },
+                        "mounts": [
+                            {
+                                "mountPath": "<path1>"
+                            },
+                            {
+                                "mountPath": "<path2>"
+                            }
+                        ]
+                    }
+                }
```

Comment:
"Only 'hostPath' supported yet." -> "Only 'hostPath' is supported."

Add more about the docstring:

1. If we just want to mount a single volume, could we write: `"mounts": {"mountPath": "<path1>"}`?
2. will we resolve the environment varaibles if `$` exists in the "path" or "mountPath"?

Reply:
1. Yes. you can. I have updated the doc.
2. Thanks for your advice. I think there is some confusion for users if we support parsing env variables because it occurs in coordinator pod, instead of python client. If we do that, we also have to support passing env in session params.